src/lamplib/src/genny/tasks/run_tests.py

```diff
@@ -163,10 +163,14 @@ def out(cmd_output: str) -> bool:
 # See the logic in _setup_resmoke.
 # These are the "Binaries" evergreen artifact URLs for mongodb-mongo compile tasks.
 # The binaries must be compatible with the version of the mongo repo checked out in use for resmoke,
-# which is the sha "cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3" mentioned below.
+# which is the sha "6e2dcc5a39eb2de9d2e8209271115c078ae16470" mentioned below.
```

Comment:
Should we use a release version? That would minimize the chances of random server bugs being in it, since the release process does some good testing.

Reply:
Looking into https://github.com/mongodb/mongo/commits/r6.0.0 "e61bf27" is the release version with no actual code changes or evergreen runs. "6e2dcc5" (which is currently being used) is the next commit. Is there another release version?

Reply:
Added official 6.0 build to canned artifacts